tests/helpers/__init__.py

```diff
@@ -18,5 +18,13 @@
 # on the ilastik web site at:
 #		   http://ilastik.org/license.html
 ###############################################################################
-from shellGuiTestCaseBase import ShellGuiTestCaseBase
-import mainThreadHelpers
+submodule_dir = '/Users/saurabh/miniconda3/envs/ilastik-py3/ilastik-meta'
+# Add all submodules to the PYTHONPATH
+from . import expose_submodules
+expose_submodules.expose_submodules(submodule_dir)
+
+#import sys
+#print(sys.path)
+
+from .shellGuiTestCaseBase import ShellGuiTestCaseBase
+from . import mainThreadHelpers
```

Comment:
Actually, the whole way of relative imports in "tests".
I will be really interested to learn the strategy you might define for this.